packages/bpk-component-map/readme.md

```diff
@@ -0,0 +1,60 @@
+# bpk-component-map
+
+> Backpack map component.
+
+## Installation
+
+```sh
+npm install bpk-component-map --save-dev
+```
+
+## Usage
+
+```js
+import React from 'react';
+import BpkMap, { BpkMapMarker } from 'bpk-component-map';
+
+const MAP_URL = 'https://maps.googleapis.com/maps/api/js?v=3.exp&libraries=geometry,drawing,places';
+
+export default () => (
+  <BpkMap
+    googleMapURL={MAP_URL}
+    loadingElement={<div />}
+    containerElement={<div style={{ height: '400px' }} />}
+    mapElement={<div style={{ height: `100%` }} />}
+    defaultZoom={15}
+    defaultCenter={{
+      lat: 27.9881,
+      lng: 86.925,
+    }}
+    options={{
+      zoomControl: false,
+      dragEnabled: false,
+    }}
+  >
+    <BpkMapMarker
+      position={{
+        lat: 27.9881,
+        lng: 86.925,
+      }}
+      mapPaneName="overlayMouseTarget"
+    >
+      Everest summit
```

Comment:
Love this

Reply:
Thought you'd like that haha. I originally did Buckingham Palace but then I thought I should be less colonial about it